.gitignore

```diff
@@ -1,9 +1,11 @@
 .svn/
+.vs/
 bin/
```

Comment:
Is this (and other changes to .gitignore) related to this PR?

Reply:
No, buts its in a separate commit, I was forcing myself to use MS VisualStudio to code this, and it puts the CMake output into the source-code directory, also a .vs folder for its own files.